src/auth/Owned.sol

```diff
@@ -16,9 +16,16 @@ abstract contract Owned {
 
     address public owner;
 
-    modifier onlyOwner() virtual {
-        require(msg.sender == owner, "UNAUTHORIZED");
+    /*//////////////////////////////////////////////////////////////
+                            CUSTOM ERRORS
```

Comment:
did you generate these w/ https://github.com/transmissions11/headers

Reply:
Negative I just use this syntax wherever I can these days since looking @ solmate / yield as example codebases.

Reply:
mind checking it against header's output? we use it throughout the codebase and want to ensure consistency